Makefile.in

```diff
@@ -177,7 +177,7 @@ build-doc:
 build-man: docs/$(PROG_NAME).1.gz
 
 docs/$(PROG_NAME).1.gz: docs/$(PROG_NAME).1
-	$(call __silent,GZIP)gzip -c9 $< >$@
+	$(call __silent,GZIP)gzip -c9n $< >$@
```

Comment:
I am curious how this affected reproducibility - `-n` claims to save the original file name, which, if the build was configured in the same way, would be the same.

Reply:
man 1 gzip says
> When  compressing,  do  not save the original file name and timestamp by default.

and timestamp is the important portion here. It is OK as long as input files come straight from the tarball, but if patches are applied or autoconf magic creates them, it varies across builds (bytes 4-7 in the .gz file)

Reply:
see e.g.
```bash
echo | gzip -c | od -tx1
```

Reply:
Ah, so it also embeds a timestamp - forgot about that. Thanks for reminding.